plugin/material-design.php

```diff
@@ -3,7 +3,9 @@
  * Plugin Name: Material Design
  * Plugin URI: https://github.com/material-components/material-design-for-wordpress
  * Description: The official Material Design plugin for WordPress. Customize your site’s navigation, colors, typography, and shapes, use Material Components, and choose from over 1,000 Google Fonts and Material Design icons. From the team behind Google’s open-source design system.
- * Version: 0.3.1
+ * Version: 0.2.1
```

Comment:
Should this be 0.3.1 until we decide to bump to 0.4 or 0.3.2? @pbearne

Reply:
The version numbers (and message text) here will need to adjusted when we do the next release